include/openssl/ec.h

```diff
@@ -1342,6 +1342,10 @@ void EC_KEY_METHOD_get_verify(const EC_KEY_METHOD *meth,
                                 EVP_PKEY_OP_DERIVE, \
                                 EVP_PKEY_CTRL_GET_EC_KDF_UKM, 0, (void *)(p))
 
+# define EVP_PKEY_CTX_set_sm2_uid(ctx, uid) \
```

Comment:
Since the UID is tied to a particular `PKEY_CTX` I assumed the application already knows what value it set, but I suppose symmetric does argue for a getter.

Reply:
Done